### DISCUSSION TITLE
Feature request: possibility to paste image into a note

### DISCUSSION THREAD

**Discussion by shalak** [karakeep-app/karakeep#204] [Ideas]
Would be great to be able to simply cmd+v a picture that I have in a clipboard into a note.

**Comment by MohamedBassem** [comment]
@shalak do you want to inline images in notes, or do you want to be able to create a new "image" type bookmark by pasting in the editor?

**Reply by shalak** [reply]
The latter. I understand that inline images in notes is a completely different feature, but Hoarder already has the note-as-an-image feature, you just need to drag&drop instead of pasting to create it.

**Reply by MohamedBassem** [reply]
Yeah, just pasting in the editor is easy. As a matter of fact, @AhmadMuj already implemented it in https://github.com/hoarder-app/hoarder/pull/89/commits/d43df95c3a55bc2c13bbb96ff70e9034c05fc9a1 but somehow we forgot to merge it as it was bundled with another feature. We should definitely add it back :)

**Comment by ahmadmucom** [comment]
Hey
I have created a new PR with the desired feature. 

https://github.com/hoarder-app/hoarder/pull/228

**Comment by MohamedBassem** [comment]
Just merged @AhmadMuj's PR, this will be available in the next release. Thanks @AhmadMuj!

**Comment by filviu** [comment]
I'd like to resurect this thread with a question - would it be possible to have  a note type "multiple images" ? I know I can drom an image to create a note and then (with the latest version) I can attach more images to the note, but inline it's only showing the first image...
